src/toHaveStyleRule.js

```diff
@@ -20,10 +20,18 @@ const getClassNames = received => {
 const hasClassNames = (classNames, selectors) =>
   classNames.some(className => selectors.includes(`.${className}`))
 
-const getRules = (ast, classNames) =>
-  ast.stylesheet.rules.filter(
+const getRules = (ast, classNames, options) => {
+  let rules = ast.stylesheet.rules
+  if (options.media) {
+    rules = rules
```

Comment:
Because I already set the default to an empty object, it would always be true. :scream:
```js
function toHaveStyleRule(received, property, value, options = {}) {
```
or change its default to ```false``` I'm okay, or remove the default :joy:, if you like.